modules/cram/templates/manta_call.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -euo pipefail
+
+create_bed () {
+  echo -e "!{bedContent}" > "!{bed}"
+  ${CMD_BGZIP} -c "!{bed}" > "!{bedGz}"
+  ${CMD_TABIX} "!{bedGz}"
+}
+
+config_manta () {
+    local args=()
+    args+=("/opt/manta/bin/configManta.py")
```

Comment:
https://molgenis.gitbooks.io/ngs_dna/content/ngs-protocols.html mentions `--exome`. is that an option we should use as well? or make configurable?

Reply:
done - change is on another line so not picked up by github in this view